freedomofpress/securedrop-workstation
# Remove sd-export-usb persistent USB device attachment

Removes the configuration of a single USB device persistently attached to `sd-export-usb`, in preparation for automatic attachment of export devices. Until that happens, the Qubes Devices tool should be used to attach them before exporting.

Fixes #320.

## Testing

- In your `dom0` clone of `securedrop-workstation`, run `make remove-sd-export` to remove your current configuration.
- Check out this branch in your source repository.
- Back in `dom0`, run `make clone` to pull these changes.
- (Optional) Remove the "usb" attribute from your `config.json`.
- Run `make sd-export` to recreate the export domains.
- Start `sd-export-usb` with Qube Manager.
- Attach a disk export device to `sd-export-usb` with Qubes Devices.
- Export something from the client, or if you have a disk export archive handy, just export it with `qvm-open-in-vm sd-export-usb "filename"`.
- The export should succeed.
